scripts/minikube.sh

```diff
@@ -50,16 +50,28 @@ function detect_minikube() {
 
 # install minikube
 function install_minikube() {
+    local mku_version latest_version
+    mku_version=$(${minikube} update-check 2> /dev/null | grep "LatestVersion" || true)
+    latest_version=$(echo "${mku_version}" | cut -d' ' -f2)
+    if [[ -z "${latest_version}" ]]; then
+        # skip: update-check failed for some reason, lets continue with what we have
+        latest_version=${MINIKUBE_VERSION}
+    fi
     if type "${minikube}" >/dev/null 2>&1; then
         local mk_version version
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
         echo "minikube already installed with ${version}"
-        return
+        MINIKUBE_VERSION=${version}
```

Comment:
Is it needed to change the `MINIKUBE_VERSION` in this function? If it contains `latest`, that should be fine, I think?

Reply:
Just feeding it with the real version number for future use `within the script`.
Here it `might` be a lower version, and at the end, once the update is done then we assign with the latest.

@nixpanic do you see any issue with this?

Reply:
This way MINIKUBE_VERSION is always set to the currently installed version rather than faking as `lastest` always.

Reply:
Because `MINIKUBE_VERSION` gets passed into the script through an environment variable, changing its value is ugly. If there is a need for using the actual version later on, it is cleaner to add a new (lowercase) variable for that.

Reply:
@nixpanic ok, updated addressing the review comments. Thanks!